grails-app/conf/Config.groovy

```diff
@@ -1711,6 +1712,12 @@ openboxes.supportLinks = [
     knowledgeBase: 'https://openboxes.helpscoutdocs.com/',
 ]
 
+// Reset an instance
+
+openboxes.resettingInstance = [
```

Comment:
Change it to:
`openboxes.resettingInstance.command = "wget https://raw.githubusercontent.com/openboxes/openboxes/develop/reset-database.sh | sh"`